### PR TITLE
Add doa-bucharest file

### LINKED ISSUE #29: doa's Workflow for adding Bucharest to the repository
## Bucharest
- [x] spam @yjuteau-processia 's inbox
- [x] Create an issue
- [x] Create a branch
- [x] Make changes
- [x] Open a PR
- [x] Collaborate
- [ ] Get approval
- [ ] Merge

### PR DESCRIPTION
**Pull request by odragu-processia**
Closes #29

### CODE REVIEW

File: doa-bucharest.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## yay a title !
+
+### Restaurants
+- Plenty
+
+### Places to go
+- Quite a few
+
+### Current temperature
+- 16 C
```
Comment (odragu-processia) [inline doa-bucharest.md:11]: 🐉

Comment (yjuteau-processia) [reply doa-bucharest.md:11]: Are they stealing wallets?

Comment (rice-ball-rolling) [inline doa-bucharest.md:11]: nice

**Review by rice-ball-rolling — COMMENTED** [review]
xoxo

File: doa-bucharest.md (hunk 1 above)
Comment (odragu-processia) [reply doa-bucharest.md:11]: 🙈

Comment (bwang-processia) [inline doa-bucharest.md:11]: Cold ❄️ !

Comment (klebel-processia) [reply doa-bucharest.md:11]: I lost mine

Comment (jaimemendozadev) [reply doa-bucharest.md:11]: Can I get any decent Mexican food in Bucharest? Just askin'. 😄

Comment (odragu-processia) [reply doa-bucharest.md:11]: You sure can :)

**Review by klebel-processia — APPROVED** [review]

**Pull request merged**